Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/PaywallPresenter.swift

```diff
@@ -64,7 +46,7 @@ struct PaywallPresenter: View {
                                customerInfo: nil,
                                condensed: true,
                                introEligibility: .producing(eligibility: introEligility),
-                               purchaseHandler: .default())
+                                                            purchaseHandler: .default())
```

Comment:
not sure what happened with indentation here?

Reply:
Oops! I saw this after I merged it. Will fix later, I guess? Or a new PR...?